enterprise/backend/src/metabase_enterprise/sso/integrations/sso_settings.clj

```diff
@@ -105,19 +105,16 @@ on your IdP, this usually looks something like `http://www.example.com/141xkex60
 
 (defsetting saml-attribute-email
   (deferred-tru "SAML attribute for the user''s email address")
-  :default "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress"
```

Comment:
Removed these defaults as they caused issues. Should we keep them and filter out in the code?